universal-application-tool-0.0.1/app/views/applicant/ApplicantProgramSummaryView.java

```diff
@@ -152,8 +154,8 @@ private ContainerTag renderQuestionSummary(
 
     ContainerTag answerContent;
```

Comment:
Unrelated, optional: You can put a final here to ensure all parts of the if/else set this before the first read and only once.  A neat use of final.

Reply:
I didn't know you could do that! Super cool